.github/workflows/build.yml

```diff
@@ -78,20 +78,20 @@ jobs:
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
         registry: docker.pkg.github.com
-        repository: vkuznet/RucioTracers/RucioTracers
+        repository: dmwm/ruciotracers/ruciotracers
```

Comment:
What difference the github repo name "RucioTracers" vs "ruciotracers" make?

Reply:
the repositories for upload should have lower-case letters (I found it via build failures)